src/timeseriesflattener/feature_spec_objects.py

```diff
@@ -292,17 +316,19 @@ def is_dichotomous(self) -> bool:
 
 
 class MinGroupSpec(BaseModel):
-    """Minimum specification for a group of features, whether they're looking
-
-    ahead or behind.
+    """Minimum specification for a group of features, whether they're looking ahead or behind.
 
     Used to generate combinations of features.
     """
 
     values_loader: list[str]
-    # Loader for the df. Tries to resolve from the resolve_multiple_nfs registry,
+    # Loader for the df. Tries to resolve from the data_loaders registry,
     # then calls the function which should return a dataframe.
 
+    values_df_dict: Optional[list[str]] = None
+    # List of strings that correspond to the key of a dataframe in a dictionary
```

Comment:
We need a test that shows that `create_combinations` still works when we've added multiple ways of specifying keys.